ros/src/serow_ros/src/serow_driver.cpp

```diff
@@ -0,0 +1,166 @@
+#include "pinocchio/fwd.hpp" // Always first include to avoid boost related compilation errors
+#include <functional>
+#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/WrenchStamped.h>
+#include <map>
+#include <queue>
+#include <ros/ros.h>
+#include <sensor_msgs/Imu.h>
+#include <sensor_msgs/JointState.h>
+#include <serow/Serow.hpp>
+
+class SerowDriver {
+public:
+    SerowDriver(const ros::NodeHandle& nh)
+        : nh_(nh) {
+        // Load parameters from the parameter server
+        if (!nh_.getParam("joint_state_topic", joint_state_topic_)) {
+            ROS_ERROR("Failed to get param 'joint_state_topic'");
+        }
+        if (!nh_.getParam("base_imu_topic", base_imu_topic_)) {
+            ROS_ERROR("Failed to get param 'base_imu_topic'");
+        }
+        if (!nh_.getParam("force_torque_state_topics", force_torque_state_topics_)) {
+            ROS_ERROR("Failed to get param 'force_torque_state_topics'");
+        }
+        if (!nh_.getParam("config_file_path", config_file_path_)) {
+            ROS_ERROR("Failed to get param 'config_file_path'");
+        }
+
+        // Initialize SERoW
+        serow_ = serow::Serow(config_file_path_);
+        pose_estimate_.header.frame_id = "base_link";
+
+        // Create subscribers
+        joint_state_subscription_ = nh_.subscribe(joint_state_topic_, 1000, &SerowDriver::joint_state_topic_callback, this);
+        base_imu_subscription_ = nh_.subscribe(base_imu_topic_, 1000, &SerowDriver::base_imu_topic_callback, this);
+        base_state_publisher_ = nh_.advertise<geometry_msgs::PoseStamped>("/serow/base_estimate", 100);
```

Comment:
call this /serow/base/odom and use /serow/base/twist (up for discussion) for velocity 
similarly for com /serow/com/odom and /serow/com/twist